src/app/portlets/dot-edit-page/layout/dot-edit-layout-grid/dot-edit-layout-grid.component.ts

```diff
@@ -146,4 +175,52 @@ export class DotEditLayoutGridComponent implements OnInit {
         }
         return rowArray;
     }
+
+    private transformDataToDisplayOnGrid(resp): any {
+        let grid = [];
+        resp.layout.body.rows.forEach((row, rowIndex) => {
+            row.columns.forEach(column => {
+                grid.push({
+                    containers: column.containers.map(
+                        containerId => (resp.containers[containerId] ? resp.containers[containerId].container : containerId)
+                    ),
+                    config: Object.assign({}, DotEditLayoutGridComponent.NEW_ROW_TEMPLATE, {
+                        sizex: column.width,
+                        col: column.leftOffset,
+                        row: rowIndex + 1
+                    })
+                });
+            });
+        });
+        return grid;
+    }
+
+    public getLayoutBody(): LayoutBody {
+        return this.transformDataToLayoutBody();
+    }
+
+    private transformDataToLayoutBody(): LayoutBody {
```

Comment:
This method should be something like: ```get...``` no matter what transformation does... and the end who call it it's getting it.